src/Backend/Modules/MediaGalleries/Layout/Templates/MediaGalleryAdd.html.twig

```diff
@@ -0,0 +1,96 @@
+{% extends '/Layout/Templates/base.html.twig' %}
+
+{% block content %}
+  {{ form_start(form) }}
+  {{ block('media_groups_helper') }}
+  <div class="row fork-module-content">
+    <div class="col-md-12">
+      {{ form_row(form.title, {'attr': {'class': 'form-control title'}}) }}
+    </div>
+  </div>
+
+  <div class="row fork-module-content">
+    <div class="col-md-12">
+      <div role="tabpanel">
+        <ul class="nav nav-tabs" role="tablist">
+          <li class="active" role="presentation">
+            <a href="#tabContent" aria-controls="tabContent" role="tab" data-toggle="tab">{{ 'lbl.Content'|trans|ucfirst }}</a>
+          </li>
+        </ul>
+
+        <div class="tab-content">
+
+          <div role="tabpanel" class="tab-pane active" id="tabContent">
+            <div class="row">
+              <div class="col-md-8">
+                <div id="publishOptions" class="panel panel-default">
```

Comment:
I know, but if the base layout is the same then you can copy that and add your extra's within that structure